src/ViewHelpers/DebugViewHelper.php

```diff
@@ -70,6 +64,83 @@ public function render() {
 			return (is_object($expressionToExamine) ? get_class($expressionToExamine) : gettype($expressionToExamine));
 		}
 
-		return var_export($expressionToExamine, TRUE);
+		$html = $this->arguments['html'];
```

Comment:
This is really tricky here. We either would need to disable output escaping (we encode for HTML ourselves) or let it enabled, so that the interceptor can do so where required. However having both in one view helper is impossible ind imho not optimal design.

Output is either too much encoded or to less